Library/ViewModels/UpdateDraftViewModel.swift

```diff
@@ -212,7 +212,7 @@ public final class UpdateDraftViewModel: UpdateDraftViewModelType, UpdateDraftVi
 
     self.showRemoveAttachmentConfirmation = addedAttachments
       .takePairWhen(self.attachmentTappedProperty.signal)
-      .map { attachments, id in attachments.filter { $0.id == id }.first }
+      .map { attachments, id in attachments.first { $0.id == id } }
```

Comment:
I noticed that in some places we use `.first(where: { .... })` and in others  simply `.first { ... }`. Should we adopt only one to keep it consistent? I don't have strong feelings about it though.

Reply:
Yeah that's intentional, when using `first` in a `guard` statement the compiler isn't able to figure it out unless we add the `where:` argument and parentheses.